src/plugins/discover/public/application/angular/doc_table/components/_table_header.scss

```
@@ -1,3 +1,6 @@
+.kbnDocTableHeader th{
```

```suggestion
.kbnDocTableHeader {
```

This is safe against the whole header. You don't want to use this against a naked dom element.

---

thx @snide !